elasticsearch_dsl/utils.py

```diff
@@ -496,51 +525,52 @@ def __get_field(cls, name):
                     return cls._index._mapping[name]
                 except KeyError:
                     pass
+        return None
```

Comment:
Should this only apply to `except KeyError`? The try clause already returns. (Since this is only needed to help developers reason about the code, after all)

```suggestion
          return None
```